src/main/java/ObjectRepositorys/RegistretionPage.java

```diff
@@ -0,0 +1,16 @@
+package ObjectRepositorys;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+
+public class RegistretionPage {
+	public void TC() {
+		System.out.println("I have added Registetion page");
+	}
```

Comment:
you have to add some remaining remaining xpath of element with its method

Reply:
Hello VIjayDYaligar I have done changes as per your suggetion please review once.